ballerina/modules/wssec/encryption.bal

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/crypto;
+import ballerina/jballerina.java;
+
+class Encryption {
```

Comment:
I think these classes need to be marked as isolated

Reply:
Fixed in [8f8331c](https://github.com/ballerina-platform/module-ballerina-soap/pull/103/commits/8f8331c91d2f878d38ccf4f90fdb18e055b591dc)